pgmpy/base/simple_causal_model.py

```diff
@@ -0,0 +1,140 @@
+from typing import Iterable, Optional, Union
+
+from pgmpy.base import DAG
+
+
+class SimpleCausalModel(DAG):
+    """
+    A specialized DAG class for simple causal models.
+
+    This class simplifies the creation of causal graphs commonly used in causal inference,
+    where the structure consists of exposures, outcomes, covariates (confounders), mediators, and instruments.
+    It automatically adds the standard edges:
+        - Exposures -> Outcomes
+        - Covariates -> Exposures
+        - Covariates -> Outcomes
+        - Instruments -> Exposures
+        - Exposures -> Mediators
+        - Mediators -> Outcomes
+
+    Parameters
+    ----------
+    exposures: str, int, or iterable
+        The exposure variable(s). If an int is provided, the variable name will be generated as "Var_<int>".
+
+    outcomes: str, int, or iterable
+        The outcome variable(s). If an int is provided, the variable name will be generated as "Var_<int>".
+
+    covariates: str, int, iterable, or None (default: None)
+        The covariate (confounder) variable(s). If an int is provided, the variable name will be
+        generated as "Var_<int>".
+
+    mediators: str, int, iterable, or None (default: None)
+        The mediator variable(s). If an int is provided, the variable name will be generated as "Var_<int>".
+
+    instruments: str, int, iterable, or None (default: None)
+        The instrumental variable(s). If an int is provided, the variable name will be generated as "Var_<int>".
+
+    latents: iterable or None (default: None)
+        List of latent variables.
+
+    Notes
+    -----
+    If any of the `exposures`, `outcomes`, `covariates`, `mediators`, or `instruments` arguments are
+    provided as integers, they will be automatically converted to variable names in the format
+    "Var_<int>".
+
+    Examples
+    --------
+    >>> from pgmpy.base.simple_causal_model import SimpleCausalModel
+    >>> model = SimpleCausalModel(
+    ...     exposures="X", outcomes="Y", covariates="Z", mediators="M", instruments="I"
+    ... )
+    >>> model.edges()
+    OutEdgeView([('X', 'Y'), ('Z', 'X'), ('Z', 'Y'), ('I', 'X'), ('X', 'M'), ('M', 'Y')])
+        If any of the `exposures`, `outcomes`, `covariates`, `mediators`, or `instruments` arguments are
+        provided as integers, they will be automatically converted to variable names in the format "Var_<int>".
+    >>> model2 = SimpleCausalModel(
+    ...     exposures=1, outcomes=2, covariates=3, mediators=4, instruments=5
+    ... )
+    >>> model2.nodes()
+    ['Var_1', 'Var_2', 'Var_3', 'Var_4', 'Var_5']
+    >>> model2.edges()
+    OutEdgeView([
+            ('Var_1', 'Var_2'), ('Var_3', 'Var_1'), ('Var_3', 'Var_2'),
+            ('Var_5', 'Var_1'), ('Var_1', 'Var_4'), ('Var_4', 'Var_2')])
+    """
+
+    @staticmethod
+    def _to_list(var):
+        if var is None:
+            return []
+        elif isinstance(var, str):
+            return [var]
+        elif isinstance(var, int):
```

Comment:
I don't think the behavior here is correct. When an integer is passed, we should create that many variables with that role.